docs/examples/plot_pipeline_xgboost.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+.. _example-xgboost:
+
+Convert a pipeline with a XGBoost model
+========================================
+
+.. index:: XGBoost
+
+*sklearn-onnx* only converts *scikit-learn* models into *ONNX*
+but many libraries implement *scikit-learn* API so that their models
+can be included in a *scikit-learn* pipeline. This example considers
+a pipeline including a *XGBoost* model. *sklearn-onnx* can convert
+the whole pipeline as long as it knows the converter associated to
+a *XGBClassifier*. Let's see how to do it.
+
+A couple of errors might happen while trying to convert
+your own pipeline, some of them are described
+and explained in :ref:`errors-pipeline`.
+
+.. contents::
+    :local:
+
+Train a XGBoost classifier
+++++++++++++++++++++++++++
+"""
+import xgboost
```

Comment:
Can you sort the import statments? It's not very readable this way, especially with so many of them. Can you group them into two: standard library imports followed by third party imports(and sort them within the group)?